src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/Comment.ts

```diff
@@ -176,18 +178,17 @@ export var commentDetailDirective = (
     adhRecursionHelper,
     $window : Window,
     $q : angular.IQService,
+    $timeout,
     $translate
 ) => {
     var _update = update(adhHttp, $q);
     var _postEdit = postEdit(adhHttp, adhPreliminaryNames);
 
-    var link = (scope : ICommentResourceScope, element, attrs, column? : AdhMovingColumns.MovingColumnController) => {
-        if (column) {
-            scope.report = () => {
-                column.$scope.shared.abuseUrl = scope.data.path;
-                column.toggleOverlay("abuse");
-            };
-        }
+    var link = (scope : ICommentResourceScope, element) => {
+        scope.report = () => {
+            scope.modals = new AdhResourceActions.Modals($timeout);
```

Comment:
why do you re-create `scope.modals` on every report? Wouldn't it be better to create it once (directly in `scope.report()`)?